src/ruby/yast/builtins.rb

```diff
@@ -146,9 +146,18 @@ def self.maplist(object, &block)
       end
     end
 
-    # - Removes element from a list
-    # - Remove key/value pair from a map
-    # - Remove item from term
+    # Remove *element* from a **copy** of *object*.
+    #
+    # @param object [::Hash,::Array,Yast::Term,nil] a container
+    # @param element [Object,Integer,nil]
+    #   Any key for Hash;
+    #   Integer for Array and Term;
```

Comment:
maybe document that for negative number it returns copy of container without modification, so nothing like -1 is last or like that